posts/MichelaRocchettiBlogPost/Michela_JSoC.qmd

```diff
@@ -0,0 +1,43 @@
+---
+title: "JSoC'24 @JuliaHealth"
+description: "A resumee of my project work for Julia Summer of Code"
+author: "Michela Rocchetti"
+date: "9/26/2024"
+toc: true
+categories:
+  - news
+  - code
+  - analysis
+---
+# About me 
+Hi, I am Michela, I am from Rome and I hold a Master's degree in Physics of complex systems. My interest topics are the usage of modeling and AI methodologies in healthcare.
+
+# Project description 
```

Comment:
Could you give a bit more detail about what IPUMS is, what data sources it has, and link to IPUMS itself?